packages/perseus/src/widgets/radio/choice.jsx

```diff
@@ -9,192 +9,119 @@ import {Popover, PopoverContent} from "@khanacademy/wonder-blocks-popover";
-        // that they don't think it's correct. This value does not affect
-        // scoring or other behavior; it's just a note for the user's
-        // reference.
-        crossedOut: PropTypes.bool,
-
-        // Indicates that the user has previously selected this answer. These
-        // answers may be rendered orange in review, rather than grey if
-        // incorrect.
-        previouslyAnswered: PropTypes.bool,
-
-        // A callback indicating that this choice has changed. Its argument is
-        // an object with two keys: `checked` and `crossedOut`. Each contains a
-        // boolean value specifying the new checked and crossed-out value of
-        // this choice.
-        onChange: PropTypes.func,
-
-        // Callbacks for when the user presses the up arrow or down arrow
-        // (respectively) to navigate between choices. When called, the parent
-        // finds the prev/next choice, focuses the input, and checks the radio
-        // button if appropriate. (The reason we have custom behavior is that
-        // we actually _don't_ want to auto-select crossed-out choices!)
-        goToPrevChoice: PropTypes.func.isRequired,
-        goToNextChoice: PropTypes.func.isRequired,
-    };
-
-    static defaultProps: $FlowFixMe = {
-        apiOptions: {
-            styling: {},
-        },
-        multipleSelect: false,
-        checked: false,
-        classSet: {},
-        correct: false,
-        disabled: false,
-        editMode: false,
-        onChange: function () {},
-        showRationale: false,
-        type: "radio", // seems unused?
-        pos: 0,
-        previouslyAnswered: false,
-    };
-
-    state: $FlowFixMe = {
-        isInputFocused: false,
-        isInputActive: false,
-    };
-
-    UNSAFE_componentWillUpdate(nextProps: $FlowFixMe) {
-        if (this.state.isInputFocused && nextProps.disabled) {
-            this.setState({
-                isInputFocused: false,
-            });
-        }
+function ChoiceIconWrapper(props: ChoiceIconWrapperProps) {
+    const {apiOptions, children} = props;
+    const finalStyles =
+        typeof apiOptions.styling?.radioStyleVersion === "undefined"
+            ? false
+            : apiOptions.styling.radioStyleVersion === "final";
+
+    if (!finalStyles && !apiOptions.satStyling) {
+        return null;
     }
 
-    inputRef: ($FlowFixMe) => void = (ref) => {
-        this._input = ref;
-    };
+    return children;
+}
 
-    renderOptionStatus: () => React.Node = () => {
-        const {correct, checked, crossedOut, reviewMode, previouslyAnswered} =
-            this.props;
+export type ChoiceProps = {|
+    // TODO(kevinb) use Options.propTypes from perseus-api.jsx
+    // This change will also require make sure that item-renderer.jsx and
+    // server-item-renderer.jsx have appropriate defaults for apiOptions
+    // because many of the properties on Options.propTypes are required.
```

Comment:
This comment is stale and can be removed now that we're using Flow and the `APIOptions` type.